tests/next_tests/integration_tests/feature_tests/ffront_tests/test_execution.py

```diff
@@ -34,8 +31,9 @@
     neighbor_sum,
     where,
 )
+from gt4py.next.ffront.decorator import field_operator, program, scan_operator
```

Comment:
shouldn't `field_operator` be directly under `gt4py.next`?

Reply:
Yes, it should! thanks!

Reply:
@DropD the import is still here

Reply:
should be gone now